core/middlewares/with-routes.ts

```diff
@@ -76,7 +76,7 @@ const updateRouteCache = async (pathname: string, event: NextFetchEvent): Promis
     expiryTime: Date.now() + 1000 * 60 * 30, // 30 minutes
   };
 
-  event.waitUntil(kv.set(routeCacheKvKey(pathname), routeCache));
+  event.waitUntil(kv.set(kvKey(`v3_${pathname}`, channelId), routeCache));
```

Comment:
Thoughts on moving the key version `v3` to `kvKey()`?

Reply:
Do we need `v3`? 🤔

Reply:
It's an easy way for us to change all keys if needed. I think we should keep it.